jetty-ee9/jetty-ee9-servlet/src/test/java/org/eclipse/jetty/ee9/servlet/AsyncContextTest.java

```diff
@@ -224,48 +263,86 @@ public void testDispatchAsyncContext() throws Exception
     @Test
     public void testDispatchAsyncContextEncodedUrl() throws Exception
     {
-        String request = "GET /ctx/test/hello%20there?dispatch=true HTTP/1.1\r\n" +
-            "Host: localhost\r\n" +
-            "Content-Type: application/x-www-form-urlencoded\r\n" +
-            "Connection: close\r\n" +
-            "\r\n";
+        startServer((context) ->
+        {
+            ServletHolder encodedTestHolder = new ServletHolder(new TestServlet());
+            encodedTestHolder.setInitParameter("dispatchPath", "/test2/something%25else");
+            _contextHandler.addServlet(encodedTestHolder, "/encoded/*");
```

Comment:
Just a comment, but as some of the mappings have changed, it's a bit difficult to verify the changes.

Reply:
The mappings are the same as they are in ee10.
No point having different mappings for the same test behavior.